### ISSUE TITLE
Session Token

### ISSUE THREAD

**Issue by Claiyc** [VisualGameData/VIGAD#118]

**Comment by BeierKevin** [comment]
So your client should be able to create his own token like "Abcdef12234"?

**Comment by Claiyc** [comment]
Yes, this can be considered like a password the user sets. For generation we should research proper and cryptographically secure algorithms.

**Comment by BeierKevin** [comment]
Alright we can do that, and we maybe do research it or use a library which generates secure tokes or something similar.

**Comment by BeierKevin** [comment]
For example we could use the following which will generate a UUID

npm install uuid

// Generate a random UUID
const newAccessToken: string = uuidv4();
accessToken.value = newAccessToken;

**Comment by Claiyc** [comment]
> For example we could use the following which will generate a UUID
> 
> npm install uuid
> 
> // Generate a random UUID const newAccessToken: string = uuidv4(); accessToken.value = newAccessToken;

I think crypto.randomBytes(); may be a better idea.

see https://stackoverflow.com/questions/8855687/secure-random-token-in-node-js

**Comment by BeierKevin** [comment]
Maybe not sure

**Comment by BeierKevin** [comment]
If the user is able to create his own token by writing in the textfield, what rules should this token follow?

**Comment by Claiyc** [comment]
> If the user is able to create his own token by writing in the textfield, what rules should this token follow?

min 8 characters, uppercase, lowercase, special chars, numbers
